android/app/src/main/java/com/reactnativenavigation/modal/RnnModal.java

```diff
@@ -66,11 +73,18 @@ public void onDisplayed() {
     }
 
     public void push(Screen screen) {
+        updateToolbar(screen, mToolBar);
```

Comment:
Good catch! We also miss this when pushing a screen in Activities (`BottomTabActivity` and `SingleScreenActivity`). That's why I suggested we declare `updateToolbar` in `RnnToolbar`.